src/extension.ts

```diff
@@ -222,29 +222,39 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         await launcher.runCurrentTarget();
     }));
 
-    context.subscriptions.push(vscode.commands.registerCommand('makefile.launchTargetPath', () => {
-        telemetry.logEvent("launchTargetPath");
-        return launcher.launchTargetPath();
+    context.subscriptions.push(vscode.commands.registerCommand('makefile.getLaunchTargetPah', () => {
```

Comment:
Path is misspelled "Pah" in the next few lines

Reply:
I fixed these errors but didn't realized that I didn't push an update commit. Thanks.